ssh-uuid.sh

```diff
@@ -367,8 +367,11 @@ function do_proxy {
 	{ set +x; } 2>/dev/null
 	sleep 1 # poor man's wait for the background socat process to be ready
 	[ -n "${DEBUG}" ] && set -x
+	set +e
 	socat - "PROXY:127.0.0.1:${TARGET_HOST}:${TARGET_PORT},proxyport=${SOCAT_PORT},proxy-authorization-file=${PROXY_AUTH_FILE}"
 	{ local status="$?"; [ -n "${DEBUG}" ] && set +x; } 2>/dev/null
+	set -e
+	kill $(jobs -p) && wait # shutdown background tunnel
```

Comment:
```suggestion
	kill "$(jobs -p)" 2>/dev/null && wait # shutdown background tunnel
```

The double quotes avoid a [shellcheck](https://www.shellcheck.net) warning:

```sh
$ shellcheck ssh-uuid.sh

In ssh-uuid.sh line 374:
	kill $(jobs -p) && wait # shutdown background tunnel
             ^--------^ SC2046 (warning): Quote this to prevent word splitting.

For more information:
  https://www.shellcheck.net/wiki/SC2046 -- Quote this to prevent word splitt...
```

`2>/dev/null` covers the hypothetical case where `jobs -p` produced an empty output, perhaps because the first `socat` command exited earlier than expected:

```sh
$ kill "" && wait
-bash: kill: `': not a pid or valid job spec

$ kill "" 2>/dev/null && wait
```

Alternativey, if you prefer:
```suggestion
	local pid
	pid="$(jobs -p)"
	[ -n "${pid}" ] && kill "${pid}" && wait # shutdown background tunnel
```